src/org/joml/AABBdc.java

```diff
@@ -0,0 +1,485 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2017-2020 JOML
```

Comment:
Please make this (and in all other interfaces) read `Copyright (c) 2020 JOML`
I'd like to start the interval with the year of introduction of that particular compilation unit.